app/src/main/java/com/example/musclenerds/ui/exerciseview/exercise_view.java

```diff
@@ -0,0 +1,263 @@
+        exercise19.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise20 = new Exercise();
+        exercise20.id = 6;
+        exercise20.exerciseName = "Legolas";
+        exercise20.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise21 = new Exercise();
+        exercise21.id = 7;
+        exercise21.exerciseName = "Lego Land";
+        exercise21.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        legs.exercises.add(exercise15);
+        legs.exercises.add(exercise16);
+        legs.exercises.add(exercise17);
+        legs.exercises.add(exercise18);
+        legs.exercises.add(exercise19);
+        legs.exercises.add(exercise20);
+        legs.exercises.add(exercise21);
+
+        MuscleGroup arms = new MuscleGroup();
+        arms.id = 4;
+        arms.muscleName = "Arms";
+        arms.exercises = new ArrayList<Exercise>();
+
+        Exercise exercise22 = new Exercise();
+        exercise22.id = 1;
+        exercise22.exerciseName = "Crunchy Arms";
+        exercise22.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise23 = new Exercise();
+        exercise23.id = 2;
+        exercise23.exerciseName = "Armageddon";
+        exercise23.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise24 = new Exercise();
+        exercise24.id = 3;
+        exercise24.exerciseName = "Pull Ups";
+        exercise24.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise25 = new Exercise();
+        exercise25.id = 4;
+        exercise25.exerciseName = "Army Time";
+        exercise25.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise26 = new Exercise();
+        exercise26.id = 5;
+        exercise26.exerciseName = "High Five";
+        exercise26.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise27 = new Exercise();
+        exercise27.id = 6;
+        exercise27.exerciseName = "Arm Thing";
+        exercise27.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        Exercise exercise28 = new Exercise();
+        exercise28.id = 7;
+        exercise28.exerciseName = "Arm Land";
+        exercise28.imageUrl = "http://images.esellerpro.com/2347/I/919/71/GYMDUMBELL15kga.jpg";
+
+        arms.exercises.add(exercise22);
+        arms.exercises.add(exercise23);
+        arms.exercises.add(exercise24);
+        arms.exercises.add(exercise25);
+        arms.exercises.add(exercise26);
+        arms.exercises.add(exercise27);
+        arms.exercises.add(exercise28);
+
+        muscles.add(chest);
+        muscles.add(legs);
+        muscles.add(back);
+        muscles.add(arms);
+
+        return muscles;
+    }
```

Comment:
This is the "hack" I'm talking about. We will need to replace this with exercises from our db soon.